appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -0,0 +1,120 @@
+$location=$args[0]
```

Comment:
Yep. That's definitely more descriptive.

Do you have any advice for naming variables in general? I usually name things the with the first name that comes to mind, which is usually pretty generic